cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -1381,24 +1689,38 @@ makeByronTransaction :: [TxIn]
                      -> [TxOut ByronEra]
                      -> Either (TxBodyError ByronEra) (TxBody ByronEra)
 makeByronTransaction txIns txOuts =
```

Comment:
I wouldn't bother refactoring this as it's deprecated

Reply:
I needed to extract `makeByronTransactionBodyContent`